modules/crud2.js

```diff
@@ -0,0 +1,51 @@
+// reset todo id
+export const idReset = (lists) => {
+  lists.forEach((todo, ind) => {
+    todo.id = ind + 1;
+  });
+};
+
+// logic to delete all completed todos
+export const deleteCompletedTodos = (todoListss, todoUl) => {
+  const newTodos = todoListss.filter((item) => item.completed !== true);
+  idReset(newTodos);
+  localStorage.setItem('todos', JSON.stringify(newTodos));
+
+  const elements = [...todoUl.children];
+  elements.forEach((ele) => {
+    if (ele.id === 'completed') {
+      ele.remove();
+    }
+  });
+};
+
+// functionality to mark a todo as completed
+export const selectTodo = (checkBox, todoTitle, item, itemContainer) => {
+  itemContainer.id = 'completed';
+  if (checkBox.checked) {
+    todoTitle.style.textDecoration = 'line-through';
+    item.completed = true;
+  } else {
+    todoTitle.style.textDecoration = 'none';
+    item.completed = false;
+  }
```

Comment:
- [ ] Let's work on the requirement for the checkbox user interaction. Currently, when a user clicks the checkbox, the items remain unchanged in the local storage.

![checkbox](https://user-images.githubusercontent.com/53124548/214776660-c886db80-5ead-42a9-9d77-0e968a7a1055.png)

![user-action](https://user-images.githubusercontent.com/53124548/214776825-40b39f3d-31a6-463c-9202-01709b3173ed.png)

Reply:
Hello sir 
sir I am really confused how to solve this can you guide me how to solve it